custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssueNavigator.kt

```diff
@@ -65,6 +86,59 @@ class CloudIssueNavigator(
         )
     }
 
+    override fun clickOnDetails() {
+        driver.wait(
+            elementToBeClickable(By.xpath("//*[@class = 'show-filter-details']"))
+        )
+            .click()
+    }
+
+    override fun checkSubscriptions() {
+        driver.wait(
+            and(
+                visibilityOfElementLocated(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[@class = 'search-owner']")),
+                visibilityOfElementLocated(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[contains(text(), 'Owned by')]")),
+                visibilityOfElementLocated(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[contains(text(), 'Edit permissions')]")),
+                visibilityOfElementLocated(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[contains(text(), 'Edit permissions')]"))
+            )
+        )
```

Comment:
we could simplify locators
```suggestion

        val detailsView = driver.wait(visibilityOfElementLocated(By.id('inline-dialog-filter-details-overlay'))
        driver.wait(
            and(
                visibilityOfNestedElementsLocatedBy(detailsView, By.className("search-owner")),
                ....
            )
        )
```